lib/syskit/cli/log_runtime_archive.rb

```diff
@@ -43,13 +43,16 @@ def initialize(
             # through FTP server
             #
             # @param [Params] server_params the FTP server parameters
+            # #param [Integer] min_free_space the minimum required space
+            #   in the root_dir to perform transfer
```

Comment:
Is it in bytes? I would use the same unit of space as the other commands i.e. in MB to avoid confusion